app/etl/item/clients/publishing_api.rb

```diff
@@ -0,0 +1,53 @@
+require 'gds_api/publishing_api_v2'
+
+class Item::Clients::PublishingAPI
+  PER_PAGE = 50
```

Comment:
If we have this constant `PER_PAGE`, do we also need the instance variable `@per_page`?

Reply:
Fixed in d625cb032655d13beb3378a33dca9b12cd702d81